clients/datastore/lib/google_api/datastore/v1/model/value.ex

```diff
@@ -51,3 +67,9 @@ defimpl Poison.Decoder, for: GoogleApi.Datastore.V1.Model.Value do
   end
 end
 
+defimpl Poison.Encoder, for: GoogleApi.Datastore.V1.Model.Value do
+  def encode(value, options) do
+    GoogleApi.Datastore.V1.Deserializer.serialize_non_nil(value, options)
```

Comment:
This change is also required for Datastore. We don't serialize and send nil attributes.

Reply:
This *may* require configuration per api or model.